app/models/root-store/setup-root-store.ts

```diff
@@ -90,7 +90,7 @@ export async function setupRootStore() {
     rootStore = createRootStore(env, data)
 
     // Setup Authcore
-    await env.setupAuthCore()
+    env.setupAuthCore()
```

Comment:
Don't need to wait for `env.setupAuthCore()` before `rootStore.userStore.authCore.resume()`?

Reply:
Oh the function is now sync, but what happen to the `refreshToken`?

Reply:
turns out no one is actually inputting any token as param into the function